homework_1/zad2/taxLibrary/taxLibrary.go

```diff
@@ -0,0 +1,55 @@
+package taxLibrary
+
+import (
+	"github.com/pkg/errors"
+	"math"
+)
+
+type TaxClass struct {
+	Start      int
+	End        int
+	Percentage float64
+}
+
+func checkClassesDefinitions(classes []TaxClass) error {
+
+	for n, class := range classes {
+		if n != len(classes) - 1 {
+			if class.End != classes[n+1].Start {
+				return errors.New("Marginal values are not the same.")
+			}
+		}
+		if class.Percentage < 0 {
```

Comment:
moving this as the first check makes it very clear it is always checked - this is just a minor suggestion on better readability, the way you did it is fine